### PR TITLE
Conformance criteria mk

### PR DESCRIPTION
**Pull request by markkramerus**
This PR combines upgrade to SUSHI 1.0 and the addition of a Conformance page.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,13 +1,9 @@
...
-package.json
-package-list.json
-SUSHI-GENERATED-FILES.md
+fsh-generated**
+input-cache/schemas
+input-cache/schemas-r5
+input-cache/publisher.jar
+output**
+template**
+temp**
```
Comment (masnick) [inline .gitignore:9]: I think these may need a `/`, e.g., `output/**`, assuming [this is the behavior you want](https://git-scm.com/docs/gitignore):

> A trailing "/**" matches everything inside. For example, "abc/**" matches all files inside directory "abc", relative to the location of the .gitignore file, with infinite depth.

File: input/pagecontent/experimental.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+[TumorSize]: StructureDefinition-mcode-tumor-size.html
+[TumorSize example]: Observation-mcode-tumor-size.html
+[Tumor]: StructureDefinition-mcode-tumor.html
+[Tumor example]: BodyStructure-mcode-tumor.html
+[TumorSizeUnitsVS]: ValueSet-mcode-tumor-size-units-vs.html
+
+#### Usage notes
+
+- If the tumor has been removed from the body, use `specimen` to reference a [Specimen](http://hl7.org/fhir/R4/specimen.html) resource. If the tumor is still in the body, use `focus` to reference a resource confirming to the [Tumor] profile.
+- The tumor dimensions are stored in three different components in [TumorSize]. `tumorLongestDimension` is required, and `tumorDimension2` and `tumorDimension3` are optional.
```
Comment (masnick) [inline input/pagecontent/experimental.md:102]: This is the only part of the experimental tumor size profile I saw that snuck in. This entire file can be safely deleted...it’s in the experimental tumor size branch as well.

File: input/pagecontent/conformance.md [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+<!-- conformance.md {% comment %}
+*****************************************************************************************
+*                            WARNING: DO NOT EDIT THIS FILE                             *
+*                                                                                       *
+* This file is generated by SUSHI. Any edits you make to this file will be overwritten. *
+*                                                                                       *
+* To change the contents of this file, edit the original source file at:                *
+* fhir-mCODE-ig\input\pagecontent\conformance.md                                        *
+*****************************************************************************************
+{% endcomment %} -->
```
Comment (masnick) [inline input/pagecontent/conformance.md:80]: It looks like this is getting added in multiple times by a script (SUSHI bug?). We should manually clean up until the bug is fixed.

**Review by masnick — COMMENTED** [review]
I’ve noted a few minor issues in the comments on this PR.

**Review by mlterryMitre — APPROVED** [review]

**Review by mlterryMitre — CHANGES_REQUESTED** [review]
There are merge conflicts with CancerRelatedMedicationStatement.fsh.

File: input/pagecontent/conformance.md (hunk 3 above)
Comment (masnick) [reply input/pagecontent/conformance.md:80]: [This is a SUSHI bug](https://github.com/FHIR/sushi/issues/626), and is fixed by https://github.com/FHIR/sushi/pull/631 (confirmed).

I'm manually fixing this for now....we just need to be careful to continue to delete until the SUSHI binary is updated.

**Comment by markkramerus** [comment]
Made those fixes and recommitted.

**Comment by markkramerus** [comment]
Updated from Master. There should be no merge conflicts.

**Pull request merged**